From 970235c647f1f7c5ec90196e33941f64acba42e8 Mon Sep 17 00:00:00 2001
From: Francesco Romani <fromani@redhat.com>
Date: Thu, 12 Feb 2026 21:36:37 +0100
Subject: [PATCH 1/3] buildroot: bump to latest patch release

this allows compilation with gcc >= 14 (e.g. fedora 43).
Being a patch release, we pull in only stable fixes,
which should be compatible with the stability promise.

Signed-off-by: Francesco Romani <fromani@redhat.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8fd25f6ca..721ae1fb0 100644
--- a/Makefile
+++ b/Makefile
@@ -43,7 +43,7 @@ export GOTOOLCHAIN := go$(GO_VERSION)
 GO_K8S_VERSION_PREFIX ?= v1.35.0
 
 INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
-BUILDROOT_BRANCH ?= 2025.02
+BUILDROOT_BRANCH ?= 2025.02.10
 GOLANG_OPTIONS = GOWORK=off GO_VERSION=$(GO_VERSION)
 BUILDROOT_OPTIONS = BR2_EXTERNAL=../../deploy/iso/minikube-iso $(GOLANG_OPTIONS)
 REGISTRY ?= gcr.io/k8s-minikube
-- 
2.53.0

